src/js/types.ts

```diff
@@ -241,41 +236,23 @@ export interface FSType {
   ) => FSNode) & {
     major: number;
   };
-  closeStream: (fd: number) => void;
-  open: (path: string, flags: string | number, mode?: number) => FSStream;
-  makedev: (major: number, minor: number) => number;
-  mkdev: (path: string, dev: number) => FSNode;
-  filesystems: any;
-  stat: (path: string, dontFollow?: boolean) => any;
-  readdir: (path: string) => string[];
-  isDir: (mode: number) => boolean;
-  isMountpoint: (mode: FSNode) => boolean;
   lookupPath: (
     path: string,
     options?: {
       follow_mount?: boolean;
     },
   ) => { node: FSNode };
-  isFile: (mode: number) => boolean;
-  writeFile: (path: string, contents: any, o?: { canOwn?: boolean }) => void;
-  chmod: (path: string, mode: number) => void;
-  utime: (path: string, atime: number, mtime: number) => void;
-  rmdir: (path: string) => void;
-  mount: (type: any, opts: any, mountpoint: string) => any;
-  unmount: (mountpoint: string) => any;
-  write: (
-    stream: FSStream,
-    buffer: any,
-    offset: number,
-    length: number,
-    position?: number,
-  ) => number;
-  close: (stream: FSStream) => void;
-  ErrnoError: { new (errno: number): Error };
+  open: (path: string, flags: string | number, mode?: number) => FSStream;
+  filesystems: any;
+  isMountpoint: (node: FSNode) => boolean;
+  closeStream: (fd: number) => void;
   registerDevice<T>(dev: number, ops: FSStreamOpsGen<T>): void;
-  syncfs(dir: boolean, oncomplete: (val: void) => void): void;
+  writeFile: (path: string, contents: any, o?: { canOwn?: boolean }) => void;
 }
 
+// Omit lookupPath due to it's not compatible Pyodide lookupPath function
+export type FSType = Omit<typeof FS, "lookupPath"> & PyodideFSType;
```

Comment:
```suggestion
/**
 * @hidden
 */
export type FSType = Omit<typeof FS, "lookupPath"> & PyodideFSType;
```

Let's make this symbol hidden in the docs.